### PR TITLE
Add lines on english animal.md

### PR DESCRIPTION
**Pull request by kevintkim**
Please review this @casey-collab !

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2676)

### CODE REVIEW

File: english/animal/animal.md [hunk 1]
```diff
@@ -3,4 +3,9 @@ My worst fears have come true.  My cat is crying out for food.
...
 The hilarious thing is that i start thinking about making it my own meal.
 I remember my last meal, oh it was a dead dog 3 days ago.
-We are stepping to the verge of eating each other, human flesh oh what does it taste.
\ No newline at end of file
+We are stepping to the verge of eating each other, human flesh oh what does it taste.
+
+Halfway eating, you notice that something is wrong. Where is the meatiness? The tinge
+of iron? Why is it so... sweet? 
+
+To your horror, your dog, your cat, and you yourself are [marshmallow.](english/marshmallow.md)
```
Comment (casey-collab) [inline english/animal/animal.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**